src/components/Button/README.md

```diff
@@ -289,3 +289,120 @@ Use when a button has been pressed and the associated action is in progress.
 
 - To learn how to combine or lay out multiple buttons, [use the button group component](/components/actions/button-group)
 - To embed an action into a line of text, [use the link component](/components/navigation/link)
+
+---
+
+## Accessibility
```

Comment:
This section is quite long for the button in particular. I'd love any thoughts on whether it should be presented differently. There's discussion about the formatting in the PR thread for supporting the content (https://github.com/Shopify/polaris-styleguide/pull/2493).